source/languages/en/riak/dev/taste-of-riak/object-modeling-golang.md

```diff
@@ -0,0 +1,543 @@
+---
+title: "Taste of Riak: Object Modeling with Go"
+project: riak
+version: 1.4.8+
```

Comment:
Yep! As long as it doesn't involve Yokozuna search or Riak Security, pretty much everything is 1.4 compatible.